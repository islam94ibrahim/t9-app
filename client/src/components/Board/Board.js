import React from 'react';

import styles from './Board.module.css';

const Board = (props) => {
  return (
    <div className={styles.Board}>
      <h2>Words</h2>
      <div>{props.children}</div>
    </div>
  );
};

export default Board;
