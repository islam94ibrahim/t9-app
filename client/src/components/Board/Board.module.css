.Board {
  margin: auto;
  max-width: 500px;
}

.Board h2 {
  text-align: center;
  margin-top: 50px;
  margin-bottom: 15px;
}

.Board div {
  padding: 20px;
  min-height: 50px;
  font-size: 20px;
  border: 1px solid #ccc;
}
