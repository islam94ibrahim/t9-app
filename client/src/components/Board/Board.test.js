import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Board from './Board';

configure({ adapter: new Adapter() });

describe('<Board />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Board />);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
