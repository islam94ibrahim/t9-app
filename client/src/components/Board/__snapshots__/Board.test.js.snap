// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Board /> renders correctly 1`] = `"<div class=\\"Board\\"><h2>Words</h2><div></div></div>"`;
